JEKScrollableSectionCollectionViewLayout.h

```diff
@@ -71,5 +74,9 @@ extern NSString * const JEKCollectionElementKindSectionBackground;
 - (void)collectionView:(UICollectionView *)collectionView layout:(JEKScrollableSectionCollectionViewLayout *)layout sectionDidEndDecelerating:(NSUInteger)section;
 @end
 
+@protocol JEKSectionStyleDelegate <UICollectionViewDelegateFlowLayout>
+- (BOOL)collectionView:(UICollectionView *)collectionView layout:(JEKScrollableSectionCollectionViewLayout*)collectionViewLayout flowLayoutUsedInSection:(NSInteger)section;
```

Comment:
I think you should make this function a part of the already existing delegate protocol (`JEKCollectionViewDelegateScrollableSectionLayout`) instead of creating a separate one. Will make it easier to find/use!

Also it should probably be named
```
- (BOOL)collectionView:(UICollectionView *)collectionView layout:(JEKScrollableSectionCollectionViewLayout *)collectionViewLayout shouldUseFlowLayoutInSection:(NSInteger)section;
```

and include a short documentation string above it. Use `scrollViewConfigurationForSection:` as en example!